packages/contracts/contracts/EpochKeyLiteProofVerifier.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IUnirep} from './interfaces/IUnirep.sol';
+import {IVerifier} from './interfaces/IVerifier.sol';
+
+import {EpochKeyProofVerifier} from './EpochKeyProofVerifier.sol';
+
+contract EpochKeyLiteProofVerifier is IUnirep {
```

Comment:
This contract shouldn't implement `IUnirep`. `IUnirep` defines storage variables for the unirep contract - the verifier doesn't need this storage.

Reply:
Ah I see you imported to get access to the errors we define in `IUnirep`. Instead you should define something like `IProofVerifier` or `IBaseVerifier` that defines the errors you need as well as maybe `SNARK_SCALAR_FIELD`

Reply:
Also the structs `EpochKeySignals` and `ReputationSignals` can be defined in this interface.

Reply:
Instead of making it an interface it could be a base contract that these proof verifiers inherit. Then you could define the `constructor` implementation as well (accept a single variable and assign it to a state variable `IVerifier verifier`).